applications/common/backend/src/main/resources/libs/kyanite/common/components/carousel/carousel.html

```diff
@@ -16,28 +16,64 @@
   ~
 */-->
 
-<sly data-sly-use.model="pl.ds.kyanite.common.components.models.CarouselComponent">
-  <div class="carousel">
-    <sly data-sly-test="${model.hasChildren}">
-      <div data-sly-test="${model.paginationNeeded}" class="buttons is-right">
-        <button class="button previous-page-button" disabled>
-          <span class="icon">
-            <i class="mdi mdi-arrow-left"></i>
-          </span>
-        </button>
-        <button class="button next-page-button">
-          <span class="icon">
-            <i class="mdi mdi-arrow-right"></i>
-          </span>
-        </button>
+<sly
+  data-sly-use.model="pl.ds.kyanite.common.components.models.CarouselComponent"
+>
+  <sly
+    data-sly-set.CSSclasses="${model.classes @ join=' '}"
```

Comment:
We don't have `getClasses()` method in the model class so it is supposed to be empty always.

Reply:
done. Removed this `sly` block.